boards/arm/nucleo_f767zi/doc/index.rst

```diff
@@ -0,0 +1,226 @@
+- 12-bit ADC(3) with 24 channels / 2.4 MSPS
+- 12-bit DAC with 2 channels(2)
+- True Random Number Generator (RNG)
+- 16-channel DMA
+- LCD-TFT Controller with XGA resolution
+
+Supported Features
+==================
+
+The Zephyr nucleo_f767zi board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | usb_device                          |
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | rtc                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | ADC Controller                      |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_f767zi/nucleo_f767zi_defconfig``
+
+For mode details please refer to `STM32 Nucleo-144 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+The Nucleo F767ZI board features a ST Zio connector (extended Arduino Uno V3)
+and a ST morpho connector. Board is configured as follows:
+
+- UART_2 TX/RX/RTS/CTS : PD5/PD6/PD4/PD3
+- UART_3 TX/RX : PD8/PD9 (ST-Link Virtual Port Com)
+- UART_6 TX/RX : PG14/PG9 (Arduino UART)
+- USER_PB : PC13
+- LD1 : PB0
+- LD2 : PB7
+- LD3 : PB14
+- ETH : PA1, PA2, PA7, PB13, PC1, PC4, PC5, PG11, PG13
+- USB DM : PA11
+- USB DP : PA12
+- I2C : PB8, PB9
+- PWM : PE13
+- SPI : PA4, PA5, PA6, PA7
+
+Note. The Arduino Uno v3 specified SPI device conflicts with the on-board ETH
```

Comment:
If this note is important (as it seems to be), write this as:
```
.. note::
   The Arduino Uno v3 specified SPI device conflicts with the on-board ETH
   device on pin PA7.
```

Reply:
Done, checked in. Thanks!